### DISCUSSION TITLE
Default settings behavior after changing them feels strange

### DISCUSSION THREAD

**Discussion by WHITE-developer** [solspace/craft-freeform#1579] [Questions & Answers] [answered]
### What happened?

Not sure if it is an bug, but it's behavior didn't seem logically for me:
When you have a form which has
* Ajax submit disabled
* A custom render template

and you change the default freeform settings to 
* Ajax submit enabled + lock it
* A default template + lock it + remove the custom template from source

the form gets
* a locked Ajax submit button, but still disabled
* a locked and empty form template

For me it felt like it should take over the default locked settings.
At least for the removed template, as that form now breaks.

### How can we reproduce this?

See above

### Freeform Edition

Pro

### Freeform Version

5.6.5

### Craft Version

4.12.6.1

**Comment by kjmartens** [comment, marked as answer]
Thank you for your feedback, @WHITE-developer.

I can see the confusion here. However, these are meant to be defaults for new forms, and the lock simply means that users can't update the value inside the form builder for new and existing forms.

In an ideal world, the default form builder values would be configured at the beginning and then forms would be created afterward. 🙂 

At any rate, it's likely going to cause issues for other customers if we change the behavior now, but I've noted this for future consideration. 🙂 

Thanks again for sharing your concerns!
